### PR TITLE
PMM-5988 Add fields TotalSteps and FinishedSteps

### PR DESCRIPTION
**Pull request by jprukner**
Prepare API for the progress bar.

https://petstore.swagger.io/?url=https://raw.githubusercontent.com/percona/pmm/PMM-5988-dbaas-add-progress-bar/api/managementpb/dbaas/json/dbaas.json

### CODE REVIEW

**Comment by AlekSi** [comment]
Please click this ^ CLA button

**Review by AlekSi — COMMENTED** [review]
The same as https://github.com/percona-platform/dbaas-api/pull/85#pullrequestreview-570304507

**Review by BupycHuk — APPROVED** [review]

**Comment by jprukner** [comment]
> The same as [percona-platform/dbaas-api#85 (review)](https://github.com/percona-platform/dbaas-api/pull/85#pullrequestreview-570304507)

I updated the comment accordingly.

File: api/managementpb/dbaas/dbaas.proto [hunk 1]
```diff
@@ -8,10 +8,12 @@ option go_package = "api/managementpb/dbaas;dbaasv1beta1";
 
 // RunningOperation respresents a long-running operation.
 message RunningOperation {
-  // Progress from 0.0 to 1.0; can decrease compared to the previous value.
-  float progress = 1;
+  // Finished steps of the operaion; can decrease compared to the previous value.
+  int32 finished_steps = 1;
   // Text describing the current operation progress step.
   string message = 2;
+  // Total steps needed to finish the operation; can decrease compared to the previous value.
```
Comment (AlekSi) [inline api/managementpb/dbaas/dbaas.proto:15]:
```suggestion
  // Total steps needed to finish the operation; can decrease or increase compared to the previous value.
```

Comment (jprukner) [reply api/managementpb/dbaas/dbaas.proto]: Done

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Please don't squash and force push you changes – that messes up the review, and we squash all commits on merge anyway

**Comment by jprukner** [comment]
> Please don't squash and force push you changes – that messes up the review, and we squash all commits on merge anyway

Thanks for advice, won't do it in the future PRs.

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by AlekSi** [comment]
@tiagomotasantos PTAL

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**